crates/http/src/init.rs

```diff
@@ -24,6 +30,25 @@ fn rpc_proxy() -> Router {
     Router::new().route("/", post(rpc_handler))
 }
 
+fn iron_namespace() -> Router {
+    Router::new().route("/iron/foundry_get_abi", get(foundry_get_abi_handler))
```

Comment:
can we add a foundry namespace as well so this becomes `/iron/foundry/abi`? the "get" part is implicit in the HTTP metho